_config/canonicals.yml

```diff
@@ -1,5 +0,0 @@
-# A config file that maps re-published blogposts to their origin, used in canonical links
```

Comment:
We definitely need a replacement for this; this added a canonical-url header for some posts which were re-published from other original sources. This is also missing an entry actually for the recently published [post about Vitess](https://debezium.io/blog/2020/11/04/streaming-vitess-at-bolt/). As an example, this needs to trigger the addition of the following header:

    <link href="https://medium.com/bolt-labs/streaming-vitess-at-bolt-f8ea93211c3f" rel="canonical">

Reply:
Will PR fix for this